cob_frame_tracker/src/cob_frame_tracker.cpp

```diff
@@ -236,6 +245,10 @@ void CobFrameTracker::publish_twist(ros::Duration period)
 		twist_msg.twist.angular.x = pid_controller_rot_x_.computeCommand(transform_tf.getRotation().x(), period);
 		twist_msg.twist.angular.y = pid_controller_rot_y_.computeCommand(transform_tf.getRotation().y(), period);
 		twist_msg.twist.angular.z = pid_controller_rot_z_.computeCommand(transform_tf.getRotation().z(), period);
+
+        error_msg.data.push_back(transform_tf.getRotation().x());
+        error_msg.data.push_back(transform_tf.getRotation().y());
+        error_msg.data.push_back(transform_tf.getRotation().z());
```

Comment:
The problem is:
- Before: Tab-Indentation
- Now: 4-Space-Indentation

If desired then replace everything with 4 space indentation.